### PR TITLE
Add Google client secrets to .env.example and add another test

### PR DESCRIPTION
**Pull request by BryanEliDimas**
as the title says. Just improvements I noticed while debugging something.

### CODE REVIEW

File: spec/handlers/newflow/student_signup_spec.rb [hunk 1]
```diff
@@ -69,14 +69,76 @@ module Newflow
...
         expect(User.last.receive_newsletter).to be(true)
       end
+
+      it 'outputs a user' do
+        expect(subject.outputs.user).to be_present
+      end
+    end
+
+    context 'success -- when passed-in a user_from_signed_params' do
+      # a user that already had an account, then tries to sign up using signed params (so via an LMS),
```
Comment (steveburkett) [inline spec/handlers/newflow/student_signup_spec.rb:87]: imo, context should be all u need here for comments...it should say what you're saying in the comment.  comments are rarely needed ever in code, code (esp ruby dsls like rspec, should read like prose.

File: spec/handlers/newflow/student_signup_spec.rb [hunk 2]
```diff
@@ -7,7 +7,7 @@ module Newflow
         load('db/seeds.rb')
       end
 
-      let(:result) do
+      let(:subject) do
```
Comment (steveburkett) [inline spec/handlers/newflow/student_signup_spec.rb:10]: consider a named subject, to add context.  what is this?   :student_signup? a special one? what kind of one?

File: spec/handlers/newflow/student_signup_spec.rb [hunk 3]
```diff
@@ -69,14 +69,76 @@ module Newflow
...
+        described_class.call(params: params, user_from_signed_params: user_from_signed_params)
+      end
+
+      it 'outputs a user' do
+        expect(subject.outputs.user).to be_present
+      end
+
+      it 'only creates one user' do
+        subject
```
Comment (steveburkett) [inline spec/handlers/newflow/student_signup_spec.rb:139]: subject is the object under test, it should do something.  e.g.,  signed_in_user.call (where signed_in_user is your subject).

File: spec/handlers/newflow/student_signup_spec.rb (hunk 1 above)
Comment (BryanEliDimas) [reply spec/handlers/newflow/student_signup_spec.rb:87]: True. The comment is/was more descriptive than the context. Moved the comment into the context.

File: spec/handlers/newflow/student_signup_spec.rb (hunk 2 above)
Comment (BryanEliDimas) [reply spec/handlers/newflow/student_signup_spec.rb:10]: Considered. Simply `subject` is perfectly fine IMO in this case.

File: spec/handlers/newflow/student_signup_spec.rb (hunk 1 above)
Comment (BryanEliDimas) [reply spec/handlers/newflow/student_signup_spec.rb:87]: Very true. Fixed.

File: spec/handlers/newflow/student_signup_spec.rb (hunk 3 above)
Comment (BryanEliDimas) [reply spec/handlers/newflow/student_signup_spec.rb:138]: It **does** something :)
```
        described_class.call(params: params, user_from_signed_params: user_from_signed_params)
```

Comment (steveburkett) [reply spec/handlers/newflow/student_signup_spec.rb:135]: @BryanEliDimas sorry, that wasnt my point.  my point was that the actual subject is an object (not an object and a verb).  see "RSpec's subject is a special variable that refers to the object being tested"https://stackoverflow.com/questions/38437162/whats-the-difference-between-rspecs-subject-and-let-when-should-they-be-used

Comment (BryanEliDimas) [reply spec/handlers/newflow/student_signup_spec.rb:135]: Isn't the `subject` the `described_class`, then?

Comment (steveburkett) [reply spec/handlers/newflow/student_signup_spec.rb:135]: @BryanEliDimas yeah, if it's that simple. sure.   the overall point is to use a named subject when it adds "documentation" and clarity to the test.  but of course, not to overengineer it, def didnt mean that :)

**Pull request merged**